### PR TITLE
Support Japanese TN&ITN

### PR DESCRIPTION
**Pull request by LoganLiu66**

### CODE REVIEW

File: itn/chinese/test/data/money.txt [hunk 1]
```diff
@@ -7,5 +7,4 @@
 四十五六新台币 => TWD45-6
 七百三四十欧元 => €730-40
 七百三四十马来西亚令吉 => RM730-40
-三千三百八十元五角八分 => ¥3380.58
 二十五元三毛 => ¥25.3
```
Comment (xingchensong) [inline itn/chinese/test/data/money.txt:10]: why remove this line ?

Comment (LoganLiu66) [reply itn/chinese/test/data/money.txt:10]: I made a mistake that I want to remove, which is the "三千三百八十点五八円" in Japanese ITN.

**Review by xingchensong — APPROVED** [review]

**Comment by xingchensong** [comment]
many thx!

**Comment by yuyun2000** [comment]
方便把fst分享出来吗，不然用户还要clone下来进行build

**Pull request merged**